cedar-policy/tests/public_interface.rs

```diff
@@ -198,15 +198,15 @@ fn authorize_custom_request() -> Result<(), Box<dyn Error>> {
         Response::new(Decision::Allow, [alice_view_id].into(), Vec::new())
     );
 
-    // request with Account::"jane" and an unspecified action
+    // request with Account::"jane" and the default entity
```

Comment:
Nit: "the" default makes it seem like this entity type is something special.
```suggestion
    // request with Account::"jane" and a default entity
```